### PR TITLE
XW-728 | add check for PAD slot in main page

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://wikia-inc.atlassian.net/browse/XW-728

Also fixed: https://wikia-inc.atlassian.net/browse/XW-209
Invoked 25 times: no exceptions

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/curatedcontent/CuratedMainPagePageObject.java [hunk 1]
```diff
@@ -65,6 +68,15 @@ public boolean isWikiaStatsContainerVisible() {
     return isCuratedElementVisible(wikiaStatsContainer);
   }
 
+  public boolean isMainPagePadSlotInDOM() {
+    try {
+      wait.forElementPresent(mainPagePadSlot, false);
+      return true;
+    } catch (TimeoutException e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/curatedcontent/CuratedMainPagePageObject.java]: MAJOR Either log or rethrow this exception. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/MainPageTests.java [hunk 2]
```diff
@@ -58,14 +58,13 @@ public void prepareTest() {
   }
 
   //TODO: Move logging methods to page object
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/MainPageTests.java]: INFO Complete the task associated to this TODO comment. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 8 issues:
* MAJOR 3 major
* INFO 5 info

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/18d2372cfd3590a01e0df8db56128bd8ca16f6d9/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/curatedcontent/CuratedMainPagePageObject.java#L108) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* INFO [Complete the task associated to this TODO comment.](https://github.com/Wikia/selenium-tests/blob/18d2372cfd3590a01e0df8db56128bd8ca16f6d9/src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/MainPageTests.java#L149) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)
* INFO [Complete the task associated to this TODO comment.](https://github.com/Wikia/selenium-tests/blob/18d2372cfd3590a01e0df8db56128bd8ca16f6d9/src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/MainPageTests.java#L127) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)
* MAJOR [7 duplicated blocks of code must be removed.](https://github.com/Wikia/selenium-tests/blob/18d2372cfd3590a01e0df8db56128bd8ca16f6d9/src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/MainPageTests.java) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=common-java%3ADuplicatedBlocks)
* INFO [Complete the task associated to this TODO comment.](https://github.com/Wikia/selenium-tests/blob/18d2372cfd3590a01e0df8db56128bd8ca16f6d9/src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/MainPageTests.java#L32) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)
* INFO [Complete the task associated to this TODO comment.](https://github.com/Wikia/selenium-tests/blob/18d2372cfd3590a01e0df8db56128bd8ca16f6d9/src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/MainPageTests.java#L362) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)

**Comment by vforge** [comment]
LGTM, Tested

**Pull request merged**